context.go

```diff
@@ -50,26 +50,26 @@ func NewContext(rootLevel Level) *Context {
 
 // GetLogger returns a Logger for the given module name, creating it and
 // its parents if necessary.
-func (c *Context) GetLogger(name string, labels ...string) Logger {
+func (c *Context) GetLogger(name string, tags ...string) Logger {
 	name = strings.TrimSpace(strings.ToLower(name))
 
 	c.modulesMutex.Lock()
 	defer c.modulesMutex.Unlock()
 
 	return Logger{
-		impl: c.getLoggerModule(name, labels),
+		impl: c.getLoggerModule(name, tags),
 	}
 }
 
-// GetAllLoggerLabels returns all the logger labels for a given context. The
+// GetAllLoggerTags returns all the logger tags for a given context. The
 // names are unique and sorted before returned, to improve consistency.
-func (c *Context) GetAllLoggerLabels() []string {
+func (c *Context) GetAllLoggerTags() []string {
```

Comment:
I think this is an appropriate clarification of the names of things, but I believe this means it is a breaking change, and requires a Loggo version bump, doesn't it?

Reply:
Normally yes, except that main only has v2 (to our knowledge) and it's currently a draft release.